src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -28,12 +30,26 @@ const DashboardPage = () => {
     const user = useContext(AuthContext);
     const navigate = useNavigate();
     const { data, isFetching } = useJournalEntries(user.id);
+    const [snackBarDetails, setSnackBarDetails] = useState<SnackBarDetails>({} as SnackBarDetails);
+
+    const triggerDeleteSnackBar = (deleteResults: boolean) => {
+        setSnackBarDetails({ 
+            error: deleteResults, 
+            show: true, 
+            message: deleteResults ? 'Journal entry deletion successful!' : 'Something went wrong, please try again or contact us for help.'
+        })
+    }
+
+    const dismissSnackBar = () => {
+        setSnackBarDetails({ ...snackBarDetails, show: false });
+    };
```

Comment:
Wonder if this is the snackbar that will need to be called when successfully creating a journal entry and being redirected to the dashboard...Could possibly just clear the form data and have a link in the success snackbar on the journal entry form that links to the dashboard as to make it optional